src/main/java/org/cryptomator/ui/dialogs/SimpleDialogController.java

```diff
@@ -14,6 +16,7 @@ public class SimpleDialogController implements FxController {
 	private final String cancelButtonText;
 	private final Runnable okAction;
 	private final Runnable cancelAction;
+	private final BooleanProperty cancelButtonVisible = new SimpleBooleanProperty(true);
```

Comment:
Change the type to a primitive `boolean` and initialize it in the constructor. Once build, the property never changes for a specific dialog, hence we don't need the "observability" features

Reply:
Don't forget to adjust the imports.